### ISSUE TITLE
Error when submitting a newpackage update

### ISSUE THREAD

**Issue by sanjayankur31** [fedora-infra/bodhi#381]
Hiya!

I submitted a newpackage update on bodhi just now with builds for f21, f22 and f23. The update seems to have gone through, but bodhi replied with this instead of a nice page:

```
{"status": "error", "errors": [{"location": "url", "name": "id", "description": "Invalid update id"}]}
```

I'd used a single update form, but these are the three updates that were created:
https://bodhi.fedoraproject.org/updates/anka-coder-fonts-1.100-0.3.20130409hg4348cf4ec395.fc22
https://bodhi.fedoraproject.org/updates/anka-coder-fonts-1.100-0.3.20130409hg4348cf4ec395.fc23
https://bodhi.fedoraproject.org/updates/anka-coder-fonts-1.100-0.3.20130409hg4348cf4ec395.fc22

The rhbz was updated correctly too:
https://bugzilla.redhat.com/show_bug.cgi?id=949954

Only one fedmsg was sent to #fedora-fedmsg, though (the complete message was probably truncated):
```
17:25 < fedmsg-bot> bodhi.update.request.testing -- ankursinha submitted anka-coder-fonts-1.100-0.3.20130409hg434... to testing https://admin.fedoraproject.org/updates/anka-coder-fonts-1.100-0.3.20130409hg4348cf4ec395.fc21
```

I haven't received an e-mail from bodhi about these updates yet.

**Comment by lmacken** [comment]
Hmm, this should have been fixed in #353

**Comment by ralphbean** [comment]
Yeah, I think this is actually fixed too.  @sanjayankur31, can you hard-reload your page and try this again with a new update sometime.  The JS that knows how to redirect you is up to date on the server, but I think your browser has the old one cached.

Furthermore, I think the fedmsg issue is actually okay too.  Your [datagrepper history](https://apps.fedoraproject.org/datagrepper/raw?user=ankursinha) shows all three updates being created.
